src/client/common/errors/index.ts

```diff
@@ -17,7 +17,11 @@ export function populateTelemetryWithErrorInfo(props: Partial<TelemetryErrorProp
         props.failureCategory = 'fetcherror';
     }
     props.stackTrace = serializeStackTrace(error);
-    const stdErr = error instanceof BaseError ? error.stdErr : '';
+    if (typeof error === 'string') {
```

Comment:
I can see one telemetry failing, but no errors being sent, absolutely nothing.
I suspect this is because we're rejecting with a string, instead of an error object.
This will help capture those so we can fix it.

E.g. i can see we start a notebook, but we don't even attempt to start the raw kernel, meaning something dies well before then